### ISSUE TITLE
Physics.cast returns wrong order

### ISSUE THREAD

**Issue by lefixx** [tts-community/tts-community-bug-tracker#31]
**Describe the bug**
According to documentation "Entries are in the order of being hit." Note that direction of cast is from top to bottom.

**To Reproduce**

Make a stack of three object and cast from a point of origin over them towards the table.

Doing this
```lua
hitList = Physics.cast({
    origin       = {-4, 7, 4},
    direction    = {0,-1,0},
    type         = 2,
    size = {2,2,2},
    max_distance = 6,
    debug        = true,
  })

  for i,v in ipairs(hitList) do
    print(hitList[i]["hit_object"].guid)
  end
```

**Tabletop Simulator Info (please complete the following information):**
 - Windows
 - Version v12.4.3

https://user-images.githubusercontent.com/9057787/103459711-d5b71a00-4d19-11eb-81a7-69c1c2e21000.mp4

**Comment by Benjamin-Dobell** [comment]
I think this may be a documentation error, as opposed to a bug per se. In this case TTS is just exposing the results from Unity's [`RaycastAll`](https://docs.unity3d.com/ScriptReference/Physics.RaycastAll.html) which states:

> Note that the order of the results is undefined

It's probably best to submit a PR (or issue) against the docs (https://github.com/Berserk-Games/Tabletop-Simulator-API) and if required, sort the results yourself.

**Comment by lefixx** [comment]
Fixed in v13.0 according to notes "Fixed Physics.cast not returning list in correct order"
